streams/writable-streams/aborting.js

```diff
@@ -157,16 +163,15 @@ promise_test(t => {
 
   const writer = ws.getWriter();
 
-  writer.write().catch(() => {});
+  writer.write(1).catch(() => {});
```

Comment:
This potentially could be a bug in our implementation. However it looks to be unspec'd behavior? Calling write with no chunk throws so I substituted an arbitrary value for chunk here (otherwise what are you writing?). Please advise if calling write sans chunk is desired behavior.